src/http/params.cr

```diff
@@ -132,7 +132,18 @@ module HTTP
       form_builder.to_s
     end
 
-    protected getter raw_params
+    # In contrast to `#to_h`, which returns flattened key-value pairs,
```

Comment:
Documentation of a method should not start with comparing it to another method. The first sentence should only be a very basic description.
E.g.
```cr
# Returns a `Hash` with raw parameters mapping from `String` to `Array(String)`.
#
# The return value can be supplied to `.new` to create a new instance.
#
# ```
# params = HTTP::Params.parse("foo=bar&foo=baz&qux=zoo")
# params.raw_params # => {"foo" => ["bar", "baz"], "qux" => ["zoo"]}
#
# params2 = HTTP::Params.new(params.raw_params)
# params2.to_s # => "foo=bar&foo=baz&qux=zoo"
# ```
#
# In contrast, `#to_h` returns flattened key-value pairs.
getter raw_params : Hash(String, Array(String))
```